### PR TITLE
Feature/devcontainer

### PR DESCRIPTION
**Pull request by mabiede**
- fix build of devcontainer
- handling env variable `DATABASE_URL` in tests
- move devcontainer readme do its folder
- add note to beginning of Quickstart guide

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
   "workspaceFolder": "/workspace",
   "remoteEnv": {
     "DATABASE_URL": "postgresql://admin:password@database:5432/dev",
+    "DATABASE_URL_TEST_POSTGRESQL": "postgresql://admin:password@database:5432/dev",
```
Comment (joseferben) [inline .devcontainer/devcontainer.json]: @mabiede Why are you not just using `DATABASE_URL`?

Comment (mabiede) [reply .devcontainer/devcontainer.json]: @jerben IMO the test database should not be the same as the development one. So I decided to use the same syntax as in sihl.

Comment (joseferben) [reply .devcontainer/devcontainer.json]: @mabiede Those env vars were meant to be specific to running the tests of `sihl`. I assume you have a workflow where the dev and test databases are different? Then you can always create your own alias `DATABASE_URL=postgresql://admin:password@database:5432/dev make test` no?

Comment (mabiede) [reply .devcontainer/devcontainer.json]: @jerben yes, we do. Anyway, I removed the additional variable and check only for the `DATABASE_URL` also when running the tests.

**Pull request merged**